utils/files/files.go

```diff
@@ -20,29 +20,34 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 )
 
-// This function takes a key from s3 and prepares the system for
-// download by recursively creating the path needed.
-func FilePathSetup(key *string, dwn *s3manager.Downloader) (*os.File, string) {
+// Creates a folder/file given the path in a recursive way
+// If the path finish with '/' it will create a file
+// Otherwise it will create a folder
```

Comment:
Isn't it the opposite? (directory for path/ and file for path/something) :thinking:

Reply:
Oh yeah, it's right and logic. I got confused in the explanation.